### ISSUE TITLE
Compilation error: the name 'stance' does not exist in the current context

### ISSUE THREAD

**Issue by fusion809** [OpenRA/OpenRA#20745]
## Issue Summary
When I compile OpenRA from commit 98c4eac I get the error message:

```
/data/AUR/openra-wts-git/src/OpenRA/OpenRA.Mods.Common/Traits/AutoTarget.cs(258,8): error CS0103: The name 'stance' does not exist in the current context [/data/AUR/openra-wts-git/src/OpenRA/OpenRA.Mods.Common/OpenRA.Mods.Common.csproj]
```

right after the OpenRA.Game  -> /data/AUR/openra-wts-git/src/OpenRA/bin/OpenRA.Game.dll stage of compilation runs fine.

DESCRIPTION

## Reproduction
On Arch Linux, run when commit 98c4eac is still the latest commit:

```bash
git clone https://aur.archlinux.org/openra-wts-git.git
cd openra-wts-git
makepkg -si
```

and wait and behold this error.

**Comment by pchote** [comment]
https://github.com/OpenRA/OpenRA/actions/runs/4415313261 bleed is indeed broken.

**Comment by PunkPun** [comment]
Closed by #20746
